geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/package.scala

```diff
@@ -37,6 +37,11 @@ package object accumulo {
       val SCAN_BATCH_RANGES    = SystemProperty("geomesa.scan.ranges.batch", "20000")
     }
 
+    object AccumuloMapperProperties {
+      val DESIRED_SPLITS_PER_TSERVER = SystemProperty("geomesa.mapreduce.split.count.tablet")
```

Comment:
see note at bottom about renaming these properties to "geomesa.mapreduce.splits.max" and "geomesa.mapreduce.splits.tserver.max" to be more in line with hadoop style naming